python/cudf/cudf/__init__.py

```diff
@@ -82,6 +82,12 @@
 from cudf.utils.dtypes import _NA_REP
 from cudf.utils.utils import set_allocator
 
+from .options import (
```

Comment:
Match the existing style of absolute imports.
```suggestion
from cudf.options import (
```